webapp/maintenance/source/pages/index.vue

```diff
@@ -42,7 +45,14 @@ export default {
     LocaleSwitch,
   },
   data() {
-    return { metadata, supportEmail: emails.SUPPORT }
+    return { links, metadata, supportEmail: emails.SUPPORT }
```

Comment:
why nor use emails? and select the `supportEmail` in the component?

Reply:
I just left it like it was. So this was not intentional.
Shall I change this? @ulfgebhardt 
Takes me half an hour, because of Docker etc. …

Reply:
Don't worry about it.